_posts/2021-05-14-bring-gitops-to-your-functions-with-argocd-part-2-the-missing-piece-tekton.md

```diff
@@ -0,0 +1,612 @@
+2021/05/15 15:07:38 Upstream:  => http://127.0.0.1:8080
+2021/05/15 15:07:38 Token: "$TOKEN"
+INFO[2021/05/15 15:07:38] Connecting to proxy url="wss://$WSS"
+```
+
+With your _$WSS_ and the _$TEKTON_TUTORIAL_SECRET_TOKEN_, create webhook from _Webhook_ page under the _Settings_ tab of your repository like the following:
+
+![webhook](/images/bring-gitops-to-your-functions-with-argocd-part-2-the-missing-piece-tekton/setup_webhook.png)
+
+Now everything is ready to trigger the pipeline, once we update our function code, it'll trigger the pipeline.
+
+To see everything what's happening in your cluster is opening UI's for both Argo CD and Tekton.
+You can visit _localhost:8443_ in order to connect Argo CD UI's screen but for Tekton, you need to install _Tekton Dashboard_.
+
+### 5. Install Tekton Dashboard
+
+Tekton has another great project called [Tekton Dashboard](https://github.com/tektoncd/dashboard). Tekton Dashboard is a general purpose, web-based UI for Tekton Pipelines and Tekton triggers resources. We can easily install this to our cluster and see what's goin' on our cluster. Run the following command to install Tekton Dashboard and its dependencies:
+
+```bash
+$ kubectl apply --filename https://storage.googleapis.com/tekton-releases/dashboard/latest/tekton-dashboard-release.yaml
+customresourcedefinition.apiextensions.k8s.io/extensions.dashboard.tekton.dev created
+serviceaccount/tekton-dashboard created
+clusterrole.rbac.authorization.k8s.io/tekton-dashboard-backend created
+clusterrole.rbac.authorization.k8s.io/tekton-dashboard-dashboard created
+clusterrole.rbac.authorization.k8s.io/tekton-dashboard-extensions created
+clusterrole.rbac.authorization.k8s.io/tekton-dashboard-pipelines created
+clusterrole.rbac.authorization.k8s.io/tekton-dashboard-tenant created
+clusterrole.rbac.authorization.k8s.io/tekton-dashboard-triggers created
+clusterrolebinding.rbac.authorization.k8s.io/tekton-dashboard-backend created
+service/tekton-dashboard created
+deployment.apps/tekton-dashboard created
+rolebinding.rbac.authorization.k8s.io/tekton-dashboard-pipelines created
+rolebinding.rbac.authorization.k8s.io/tekton-dashboard-dashboard created
+rolebinding.rbac.authorization.k8s.io/tekton-dashboard-triggers created
+clusterrolebinding.rbac.authorization.k8s.io/tekton-dashboard-tenant created
+clusterrolebinding.rbac.authorization.k8s.io/tekton-dashboard-extensions created
+```
+
+You can simple access to your dashboard with running the following command:
+
+```bash
+$ kubectl --namespace tekton-pipelines port-forward svc/tekton-dashboard 9097:9097
+Forwarding from 127.0.0.1:9097 -> 9097
+Forwarding from [::1]:9097 -> 9097
+
+$ open http://localhost:9097
+```
+
+I recommend you to divide screen into two part, one for Argo CD UI and one for Tekton UI to follow the process before making any changes on your repository.
+
+![follow_process](/images/bring-gitops-to-your-functions-with-argocd-part-2-the-missing-piece-tekton/follow_process.png)
```

Comment:
Rename the folder `bring-gitops-to-your-functions-with-argocd-part-2-the-missing-piece-tekton` to `2021-06-tekton`